### PR TITLE
WIP: Add Remix auth helpers

### PR DESCRIPTION
**Pull request by dijonmusters**
Will update with progress soon...

### CODE REVIEW

**Comment by danielvanc** [comment]
@dijonmusters Hey! Do you have a time frame on when you're likely to fix the conflicts to get this reviewed and merged in, please?

File: packages/remix/README.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+<!-- TODO! Come back to here! -->
+
+- Create a route for `/api/auth`
+
+- Create a `[...supabase].js` file under the newly created `auth` directory.
+
+The path to your dynamic API route file would be `/pages/api/auth/[...supabase].js`. Populate that file as follows:
+
+```js
+import { handleAuth } from '@supabase/auth-helpers-nextjs';
```
Comment (danielvanc) [inline packages/remix/README.md:42]: I'm presuming the import needs to come from `@supabase/auth-helps-remix` instead.

**Comment by dijonmusters** [comment]
Actively working on this is another branch - new PR up soon!

**Comment by dijonmusters** [comment]
> @dijonmusters Hey! Do you have a time frame on when you're likely to fix the conflicts to get this reviewed and merged in, please?

Sorry Daniel! This one got put on the back burner while we were getting out Supabase JS v2. Will open up a new PR soon with updated branch 👍

**Comment by dijonmusters** [comment]
New PR here: https://github.com/supabase/auth-helpers/pull/333

**Pull request closed without merging**